interfaces/builtin/avahi_control.go

```diff
@@ -119,9 +118,10 @@ func (iface *avahiControlInterface) StaticInfo() interfaces.StaticInfo {
 func (iface *avahiControlInterface) AppArmorConnectedPlug(spec *apparmor.Specification, plug *interfaces.ConnectedPlug, slot *interfaces.ConnectedSlot) error {
 	old := "###SLOT_SECURITY_TAGS###"
 	var new string
-	if release.OnClassic {
-		// If we're running on classic Avahi will be part
-		// of the OS snap and will run unconfined.
+	// If we're running on classic Avahi can be installed as snap or be part
+	// of the OS and it will run unconfined.
+	if isConnectedSlotSystemSlot(slot) {
```

Comment:
(nitpick) I don't really like the name - at the same time I'm not great with names. Maybe some suggestions:
- implicitSystemSlot(slot)
- ... ?

Reply:
yeah same here I don't know what's best name here I changed it already good 5 times :)
- implicitSystemSlot
- connectedImplicitSystemSlot ?

Reply:
I like those names, updated based on proposal from Jamie, thank you @jdstrand